event.go

```diff
@@ -67,22 +67,32 @@ func (e EventInfo) HasPos() bool { return e.hasPos }
 func (e EventInfo) HasDuration() bool { return e.hasDuration }
 
 // IsTouchEvent reports whether this event was triggered by a screen touch device.
-func (e EventInfo) IsTouchEvent() bool { return e.kind == keyTouch }
+// Deprecated: Use Device().IsTouch() instead.
```

Comment:
Nit: there should be an empty line before "deprecated" directive
See: https://github.com/go-critic/go-critic/issues/1453

Reply:
Ok, sure. It seems I forgot to run the linter.

Reply:
gocritic doesn't detect it yet, I just remembered seeing it a little while ago :D

Reply:
Ok, I added an empty comment line.